libs/extensions/angular/sidebar-menu/src/public-components/sidebar-header/sidebar-header.component.ts

```diff
@@ -2,16 +2,47 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'kirby-x-sidebar-header',
-  template: '<span class="sidebar-logo"><ng-content></ng-content></span>',
-  styles: [
-    `
-      .sidebar-logo {
-        display: flex;
-        align-items: center;
-        margin: 32px 0;
-        height: 50px;
+  template: `
+    <span class="sidebar-logo"><ng-content></ng-content></span>
+    <div class="action-bar">
+      <ng-content select="[slot='search']"></ng-content>
+      <ng-content select="[slot='action']"></ng-content>
+    </div>
+  `,
+  styles: `
+    :host {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      flex-grow: 1;
+    }
+
+    :host::ng-deep .sidebar-logo > img {
```

Comment:
If the logo stuff was slotted instead we could do `::ng-deep  [slot="logo"] img` (and get rid of > )which would make it more robust, and allow teams to do something like: 

```
<a href="/" slot="logo">
  <img />
</a>
```